### PR TITLE
#158749495 Resolve Heroku and Nodemon Issues

### PR DESCRIPTION
**Pull request by Onnassiz**
#### What does this PR do?
This PR resolves Heroku deploy issues and changes .nodemon config
#### Description of Task to be completed?
- delete .nodemonconfig file
- modify package.json postinstall scrip
- modify gulp config
- refractor touched files to ES6
#### How should this be manually tested?
This PR can be tested on Heroku using the review app link below.
#### What are the relevant pivotal tracker stories?
#158749495

### CODE REVIEW

**Comment by adetola-ralph** [comment]
@Onnassiz please fix the merge conflicts

File: server.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ const io = require('socket.io');
 
 // Load configurations
 // if test env, load example file
+
+
```
Comment (adetola-ralph) [inline server.js]: The extra space here is not needed

**Review by adetola-ralph — CHANGES_REQUESTED** [review]

File: server.js (hunk 1 above)
Comment (Onnassiz) [reply server.js]: Okay, @adetola-ralph. I have resolved the conflicts.

Comment (adetola-ralph) [reply server.js]: @Onnassiz Please remove the extra space on line 19

**Comment by Onnassiz** [comment]
@adetola-ralph , I have removed the empty line. Sorry I missed that earlier.

**Review by adetola-ralph — APPROVED** [review]

**Pull request merged**